build/configs/rtl8730e/hello/defconfig

```diff
@@ -0,0 +1,1437 @@
+CONFIG_RTL8730E_UART4=y
+CONFIG_RTL8730E_SERIAL_FIFO=y
+
+#
+# Realtek RTL8730E WIFI Support
+#
+CONFIG_AMEBASMART_WIFI=y
+
+#
+# Realtek RTL8730E BLE Support
+#
+# CONFIG_AMEBASMART_BLE is not set
+# CONFIG_AMEBASMART_BLE_DEBUG is not set
+# CONFIG_AMEBASMART_BLE_PERIPHERAL is not set
+# CONFIG_AMEBASMART_BLE_CENTRAL is not set
+# CONFIG_AMEBASMART_BLE_SCATTERNET is not set
+
+CONFIG_AMEBASMART_BLE=y
+CONFIG_AMEBASMART_BLE_PERIPHERAL=y
+CONFIG_AMEBASMART_BLE_CENTRAL=y
+CONFIG_AMEBASMART_BLE_SCATTERNET=y
+
+#
+# Realtek RTL8730E TrustZone Support
+#
+# CONFIG_AMEBASMART_TRUSTZONE is not set
+
+#
+# Architecture Options
+#
+# CONFIG_ARCH_NOINTC is not set
+# CONFIG_ARCH_VECNOTIRQ is not set
+# CONFIG_ARCH_HAVE_IRQTRIGGER is not set
+# CONFIG_ARCH_DMA is not set
+# CONFIG_ARCH_HAVE_IRQPRIO is not set
+CONFIG_ARCH_ICACHE=y
+# CONFIG_ARCH_ICACHE_LOCK is not set
+CONFIG_ARCH_DCACHE=y
+# CONFIG_ARCH_DCACHE_LOCK is not set
+CONFIG_ARCH_L2CACHE=y
+# CONFIG_ARCH_HAVE_COHERENT_DCACHE is not set
+CONFIG_ARCH_HAVE_ADDRENV=y
+CONFIG_ARCH_NEED_ADDRENV_MAPPING=y
+# CONFIG_ARCH_HAVE_EXTRA_HEAPS is not set
+# CONFIG_ARCH_HAVE_TEXT_HEAP is not set
+CONFIG_ARCH_HAVE_MULTICPU=y
+CONFIG_ARCH_HAVE_VFORK=y
+CONFIG_ARCH_HAVE_FPU=y
+CONFIG_ARCH_HAVE_DPFPU=y
+CONFIG_ARCH_HAVE_MMU=y
+# CONFIG_ARCH_HAVE_MPU is not set
+# CONFIG_ARCH_NAND_HWECC is not set
+# CONFIG_ARCH_HAVE_EXTCLK is not set
+# CONFIG_ARCH_HAVE_POWEROFF is not set
+# CONFIG_ARCH_HAVE_RESET is not set
+CONFIG_ARCH_USE_MMU=y
+# CONFIG_ARCH_HAVE_TESTSET is not set
+CONFIG_ARCH_HAVE_THREAD_LOCAL=y
+CONFIG_ARCH_HAVE_FETCHADD=y
+# CONFIG_ARCH_HAVE_RTC_SUBSECONDS is not set
+# CONFIG_ARCH_HAVE_SYSCALL_HOOKS is not set
+# CONFIG_ARCH_HAVE_BACKTRACE is not set
+# CONFIG_ARCH_HAVE_BOOTLOADER is not set
+CONFIG_ARCH_FPU=y
+CONFIG_ARCH_DPFPU=y
+# CONFIG_ARCH_ADDRENV is not set
+# CONFIG_PAGING is not set
+CONFIG_ARCH_STACKDUMP=y
+CONFIG_ARCH_STACKDUMP_MAX_LENGTH=0
+# CONFIG_ENDIAN_BIG is not set
+# CONFIG_ARCH_IDLE_CUSTOM is not set
+# CONFIG_ARCH_HAVE_RAMFUNCS is not set
+# CONFIG_ARCH_HAVE_RAMVECTORS is not set
+# CONFIG_SUPPRESS_INTERRUPTS is not set
+# CONFIG_SUPPRESS_TIMER_INTS is not set
+
+#
+# Board Settings
+#
+CONFIG_BOARD_LOOPSPERMSEC=22222
+# CONFIG_ARCH_CALIBRATION is not set
+
+#
+# Interrupt options
+#
+CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
+CONFIG_ARCH_INTERRUPTSTACK=4096
+# CONFIG_ARCH_HAVE_NESTED_INTERRUPT is not set
+# CONFIG_ARCH_HAVE_HIPRI_INTERRUPT is not set
+
+#
+# Boot options
+#
+# CONFIG_BOOT_RUNFROMEXTSRAM is not set
+# CONFIG_BOOT_RUNFROMFLASH is not set
+# CONFIG_BOOT_RUNFROMISRAM is not set
+CONFIG_BOOT_RUNFROMDRAM=y
+# CONFIG_BOOT_COPYTORAM is not set
+
+CONFIG_AMEBASMART_BOOT_DRAM=y
+# CONFIG_AMEBASMART_BOOT_NOR is not set
+# CONFIG_AMEBASMART_BOOT_SRAM is not set
+
+#
+# Boot Memory Configuration
+#
+CONFIG_RAM_KREGIONx_START="0x60400000,"
+CONFIG_RAM_KREGIONx_SIZE="4194304,"
+##### Need to crosscheck
+CONFIG_RAM_START=0x60300000
+CONFIG_RAM_VSTART=0x60300000
```

Comment:
Could you please generate the defconfig using menuconfig? It will help us to figure out if any Kconfig values are wrongly declared.

Reply:
Thanks for reminder. I will check Kconfig part.